zeebe/broker/src/main/java/io/camunda/zeebe/broker/system/SystemContext.java

```diff
@@ -457,6 +464,26 @@ private Set<Integer> validateFixedPartitionMembers(
     return members;
   }
 
+  // actually initializing the entities will be done in IdentitySetupInitializer.
+  // Validation is done here, only to be able to stop the application on error.
+  private void validateInitializationConfig() {
+    final AuthorizationConfigurer authorizationConfigurer =
+        new AuthorizationConfigurer(securityConfiguration.getCompiledIdValidationPattern());
+
+    final Either<List<String>, List<AuthorizationRecord>> configuredAuthorizations =
+        authorizationConfigurer.configureEntities(
+            securityConfiguration.getInitialization().getAuthorizations());
+
+    // TODO: after adding more entity types, change this, so it accounts for all violations all
+    //   together.
+    configuredAuthorizations.ifLeft(
+        (violations) -> {
+          throw new IdentityInitializationException(
+              "Cannot initialize configured entities: \n- %s"
+                  .formatted(StringUtils.join(violations, "\n- ")));
```

Comment:
❓ Do we really need StringUtils for `join`? What's wrong with the basic `String.join` 😄